jsons/Policies.json

```diff
@@ -242,7 +242,7 @@
 			},
 			{
 				"name": "Fine Arts",
-				"uniques": ["[Great Artist] is earned [33]% faster","+[20]% [Culture] from every [Museum]","+[20]% [Culture] from every [Opera House]","+[20]% [Culture] from every [Broadcast Tower]"],
+				"uniques": ["[Great Artist] is earned [33]% faster","+[20]% Culture from every [Museum]","+[20]% Culture from every [Opera House]","+[20]% Culture from every [Broadcast Tower]"],
```

Comment:
And here

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/d98b1105a0e0d4f0661c3954f3b1811a2899ab7c